app/models/order.rb

```diff
@@ -1,3 +1,9 @@
 class Order < ApplicationRecord
+	validates :email, format: { with: /\A(.+)@(.+)\z/ }, allow_blank: true
+	validates :end_point, :passengers, :start_point, presence: true
+	validates :phone, presence: true
+	validates_format_of :phone, with: /\A(0)([0-9]{9})\z/
+	validates :baggage, presence: true, allow_blank: true, allow_nil: false
+
```

Comment:
What about to add a enum for these fields?
For example:
```
  enum processing_status: {
    new: 'new',
    accepted: 'accepted',
    declined: 'declined'
  }
```
to prevent creating of orders with unacceptable statuses?